dapr-spring/dapr-spring-cloudconfig/src/main/java/io/dapr/spring/boot/cloudconfig/config/CloudConfigPropertiesDaprConnectionDetails.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.spring.boot.cloudconfig.config;
+
+import io.dapr.spring.boot.autoconfigure.client.DaprClientProperties;
+import io.dapr.spring.boot.autoconfigure.client.DaprConnectionDetails;
+
+class CloudConfigPropertiesDaprConnectionDetails implements DaprConnectionDetails {
```

Comment:
@lony2003 do we need to duplicate the DaprConnectionDetails, why are these different for the normal DaprConnectionDetails?

Reply:
No difference, just because the PropertyDaprConnectionDetails is protected and can't be used in another package.